cocos/core/data/deserialize-dynamic.ts

```diff
@@ -263,7 +263,7 @@ function compileDeserializeNative (_self: _Deserializer, klass: CCClassConstruct
             if (prop === undefined) {
                 continue;
             }
-            if (!fastMode && typeof prop !== 'object') {
```

Comment:
Why delete `!fastMode` check?

Reply:
The last time I communicated with the deserialization problem, it was caused by this.@holycanvas